src/reducers/initialState.js

```diff
@@ -454,7 +454,7 @@ export default {
                 description: "This course is primarily for sophomores and juniors preparing for the SAT.",
                 room_id: 4,
                 type: "C",
-                roster: [23, 99,],
```

Comment:
Why were rosters changed?

Reply:
The student tabs won't work because I still need to fix the enrollments object. This is a quick fix